### ISSUE TITLE
Event listener for animation complete not available.

### ISSUE THREAD

**Issue by albertfrates** [chenqingspring/react-lottie#6]
Attempting to trigger a function to re-direct the page after the animation is completed. Perhaps we are not understanding how to pull event hooks in the wrapper, however at the moment we cannot find an onComplete or similar event hook.

**Comment by chenqingspring** [comment]
@albertfrates, please try v0.2.2 with `eventListeners` support.
Thanks @blakedietz :)

**Comment by jfrux** [comment]
Events not firing in latest version of the module... anybody else having issues with it?

**Comment by aaronsmulktis** [comment]
Yes I am also trying to use the `complete` or `onComplete` eventListener callbacks and they ain't twerkin ;/~

**Comment by sanket-zeus** [comment]
Which eventName should use for animation-end callback? onAnimationEnd?

**Comment by kcbreuil** [comment]
Same @aaronsmulktis @jfrux  -___-

**Comment by orchoban** [comment]
same
